examples/color/colormap_reference.py

```diff
@@ -67,7 +67,23 @@ def plot_color_gradients(cmap_category, cmap_list):
 for cmap_category, cmap_list in cmaps:
     plot_color_gradients(cmap_category, cmap_list)
 
-plt.show()
+
+###############################################################################
+# .. _reverse-cmap:
+#
+# Reversed colormaps
+# ------------------
+#
+# Append ``_r`` to the name of any built-in colormap to get the reversed
+# version:
+
+plot_color_gradients("Original and reversed ", ['viridis', 'viridis_r'])
+
+# %%
+# The built-in reversed colormaps are generated when building the
+# `.ColormapRegistry` by calling the ``reversed`` method on the built-in
+# colormap objects. See :ref:`reversing-colormap` for more information about
+# the ``reversed`` method.
```

Comment:
```suggestion
# You can also use `.Colormap.reversed` on a colormap object to create a
# reversed version. See :ref:`reversing-colormap`.
```
How and when the builtin reversed colormaps are generated is an implementation detail, that is not relevant to our users.